src/main/java/ar/edu/uba/fi/Inmueble.java

```diff
@@ -0,0 +1,14 @@
+package ar.edu.uba.fi;
+
+public abstract class Inmueble implements Alquilable{
+
+    protected String direccion;
+
+    public Inmueble(String direccion){this.direccion = direccion;}
+
+    public abstract double alquilarPorDias(int dias);
+
+    public boolean esIgualA(Inmueble otroInmueble){return otroInmueble.tieneMismaDireccion(this.direccion);}
+
+    private boolean tieneMismaDireccion(String direccion){return this.direccion == direccion;}
```

Comment:
Ojo con usar `this.direccion == direccion`
Te puede traer problemas, es mejor usar el siguiente método
`this.direccion.equals(direccion)`

Reply:
Hola Lucas! Recién veo el comentario, perfecto lo tengo en cuenta entonces! Por curiosidad esto puede traer problemas por que tengan igual nombre el atributo y la variable? o por otra cosa?

Reply:
Depende como instancias al string esto puede compararse correctamente (el contenido del string) o incorrectamente (el puntero).
Podes hacer la prueba, crea dos strings de la forma `String str = new String("abc")` luego comparalos de las dos formas que conoces.